pycheribuild/projects/cross/cheribsd.py

```diff
@@ -67,6 +71,277 @@ def _clear_dangerous_make_env_vars():
             os.unsetenv(k)
             del os.environ[k]
 
+# XXX-AM: Replace this with CompilationTarget?
+class KernelABI(Enum):
+    NATIVE = "native"
```

Comment:
What does native even mean for CHERI when hybrid and purecap exist too?

Reply:
Probably `NOCHERI` is better here, I wanted to convey the fact that it is the target without CHERI extensions.